python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/__init__.py

```diff
@@ -28,7 +27,7 @@
 
 # module metadata
 __name__: Final[str] = "serverless-aws-lambda-sdk"
-__version__: Final[str] = version(__name__)
+__version__: Final[str] = "0.1.15"
```

Comment:
Isn't that change problematic maintenance wise? Will we now have to bump this version manually every time? We definitely should not do that

Reply:
Fair point, we'll need to do that correct. I thought we were already doing it with pyproject.toml file anyway so why not do it here as well.

I think a middle ground would be to still do it dynamically but do a lazy initialization, what do you think?

Reply:
Removing the import of `importlib_metadata` saves us 50ms that's for sure. We can choose to move it from INIT time to request time by lazy loading it, but the latency will be there. If we do it manually like in this PR, we get rid of the latency with the drawback of bumping the version manually for each release.

Reply:
I'm now evaluating the options listed at https://packaging.python.org/en/latest/guides/single-sourcing-package-version/

Reply:
> Fair point, we'll need to do that correct. I thought we were already doing it with pyproject.toml file anyway so why not do it here as well.

it's where we control package version, but it's easy to forget to update the second place where we're doing that.

We had same issue in Node.js SDK, and when release was made by other developers obviously version in other file was not updated. It's very easy to get this out of sync

Reply:
There are some other ways as linked above, I'm thinking of going with the "VERSION" file path. We'll define the version in a separate file and the project.toml has a way of reading it from that file, in the code we also read it from the file:

```
[tool.setuptools.dynamic]
version = {file = "./serverless_aws_lambda_sdk/VERSION"}
```

and in the code:

```
with open(Path(__file__).parent / "VERSION") as version_file:
    __version__ = version_file.read().strip()
```

What do you think? I think this is good such that we only have a single source of truth as well as no latency of importing a huge library just for the version reading.

Reply:
I've amended the PR with the `VERSION` file implementation, please take a look 🙏